process/release_process.md

```diff
@@ -0,0 +1,155 @@
+- [Impacted domains](#impacted-domains)
+- [How to create a release?](#how-to-create-a-release)
+  - [xt-core](#xt-core)
+  - [moulin-based products](#moulin-based-products)
+- [Split of responsibilities](#split-of-responsibilities)
+- [Example of releasing the moulin-based project](#example-of-releasing-the-moulin-based-project)
+
+Please note that this table of contents is written in Markdown and will work best in a markdown-friendly environment. Hyperlinks in the table of content won't work here. They need to be associated with the actual sections in your markdown document. The text after the '#' should match exactly the title of the section.
+
+## Overview
+
+This document describes release creation rules for the moulin development environment.
+
+The further part of the document is based on the Edge Orchestrator reference project:
+
+![Edge Orchestrator project high-level component diagram](./res/ref_project.png) 
+
+**Note!** Do not worry that a specific example is used below. The conclusions from this example have a broader context and can also be applied to other moulin-based products.
+
+## Impacted Domains
+
+Each component we develop in the context of the moulin builds should contain releases. It means that every domain has to freeze their code from time to time:
+
+- repositories with moulin configuration *.yaml files - should have their releases
+- repositories with repo manifests, west manifests, and bit-bake recipes - should have their releases
+- software component code repositories referenced by all above configurations - should have their releases
+
+But let's think of **higher levels of abstraction** and use more significant entities than git repositories.
+
+In our current practical scenario, we have an Edge Orchestrator reference project that consists of the following high-level components:
+
+- **xt-core** - a set of repositories inside the xen-troops GitHub organization that contains the core build functionality mainly as Yocto layers.
+  Note! It is up to xen-troops maintainers to define which repositories will be included in the xt-core abstraction. Currently, the most obvious candidates are listed below:
+  https://github.com/xen-troops/meta-xt-common
+  https://github.com/xen-troops/meta-xt-rcar
+  https://github.com/xen-troops/meta-xt-vhost
+  etc.
+- **AosEdge aos-rcar-gen4** - https://github.com/aoscloud/meta-aos-rcar-gen4. Moulin configuration and Yocto layers for R-Car Gen4 AosEdge distro.
+- **Cockpit meta-xt-prod-cockpit-rcar** - https://github.com/xen-troops/meta-xt-prod-cockpit-rcar. Reference demo product that implements the Cockpit multimedia system.
+
+In other words, we have **xt-core** base functionality—a reusable huge SW component consisting of multiple parts and moulin builds that want to create their releases relying on the released versions of the xt-core. Making the release means different things for these two types of entities. Let's describe the release process in these two cases.

```

Comment:
PLease add white spaces around `-` in the test `functionality—a`, so it will looks like `functionality — a`.